core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_interaction.html

```diff
@@ -75,17 +70,27 @@ <h3 ng-if="stateInteractionIdService.displayed">
         </div>
       </div>
     </form>
+
+    <form ng-if="stateInteractionIdService.displayed && !hasCustomizationArgs && customizationModalReopened">
+      This interaction has no customization options. If you'd like to delete this interaction, click the 'x' button on the top right corner.
```

Comment:
This interaction cannot be modified. To delete it, close this popup and click the 'x' button in the top-right corner.